### PR TITLE
Update readme's table of content

### LINKED ISSUE #103: Update the README's Table of Contents
The README has a lot of content added but its Table of Contents was not updated.

### PR DESCRIPTION
**Pull request by delnyn**
fix #103

### CODE REVIEW

**Comment by ericfennis** [comment]
I think you need to update your branch

**Comment by delnyn** [comment]
Oh

**Comment by delnyn** [comment]
And now?

File: README.md [hunk 1]
```diff
@@ -10,9 +10,18 @@ Lucide is a community-run fork of [Feather Icons](https://github.com/feathericon
...
+  * [CDN](#cdn)
 * [Usage](#usage)
+  * [Unpkg](#with-unpkg)
+  * [ESModules](#with-esmodules)
+    * [Options](#additional-options)
+    * [Threeshake library](#threeshake-the-library-only-use-the-icons-you-use)
+    * [Custom binding](#custom-element-binding)
   * [Figma](#figma)
-* [Contributing](#contributing)
+* [Contrbuting](#contributing)
```
Comment (ericfennis) [inline README.md:23]:
```suggestion
* [Contributing](#contributing)
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Nice work!

I saw one typo

**Pull request merged**